src/flexmock/_api.py

```diff
@@ -28,6 +28,24 @@
 RE_TYPE = type(re.compile(""))
 
 
+async def future_raise(anything: type[BaseException]) -> None:
```

Comment:
I guess we don't need these functions to be part of the public API?
```suggestion
async def _future_raise(anything: type[BaseException]) -> None:
```